src/app-middleware/middlewareFactoryList.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = [
+  /* eslint-disable global-require */
+  require('./bodyParserJsonFactory'),
+  require('./bodyParserUrlEncodeFactory'),
+  require('./compressionFactory'),
+  require('./swaggerFactory'),
+  require('./corsFactory'),
+
+  // Routes should immediately precede Error Handlers
+  require('./routesFactory'),
+  require('./unmatchedRouteHandlerFactory'),
+
```

Comment:
👍

Reply:
I didn't want to overwrite your changes when we merged back into master :)